ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/websocket/WebSocketRequestHandler.java

```diff
@@ -76,7 +77,8 @@ public void handle(
             LOG.debug("WS-RPC request -> {}", request.getMethod());
             request.setConnectionId(id);
             if (AuthenticationUtils.isPermitted(authenticationService, user, method)) {
-              future.complete(method.response(request));
+              final JsonRpcRequest fullRequest = new JsonRpcRequest(request, user);
```

Comment:
What's the distinction between request and fullRequest?

Reply:
FullRequest is now "requestContext" - and contains the request, and the user